app/javascript/src/Dashboard/styles/dashboard.scss

```diff
@@ -0,0 +1,43 @@
+$font-primary: #464646;
+
+.ApiFilter {
+
+  float: left;
+  width: auto;
+  border: 1px solid #d1d1d1;
```

Comment:
same for all hard coded width values: they are defined in https://github.com/3scale/porta/blob/dashboard-links/app/assets/stylesheets/provider/_typography.scss#L1